lib/scenic/theme.ex

```diff
@@ -0,0 +1,57 @@
+defmodule Scenic.Theme do
+  @moduledoc """
+  A Theme is a collection of settings for things like font, color pallettes, and so forth.
+  """
+  defstruct font: nil, pallette: %{}, name: nil
+
+  @type pallette :: %{
+          atom() => Scenic.Color.t()
+        }
+
+  @typedoc """
+  A Theme is a collection of settings for things like font, color pallettes, and so forth.
+
+  The available themeable properties are:
+
+  * `name` is the atom identifying a theme.
+  * `font` is a String of the font that a theme should use, or nil if no preference.
+  * `pallette` is a map mapping atoms like `:primary` or `:secondary` to a `Scenic.Color`.
+  """
+  @type t :: %__MODULE__{
+          name: :atom,
```

Comment:
We discussed removing the `name` key from the struct so that we don't have to keep the names consistent between the theme and the theme manager.

Reply:
:+1: